libafl_frida/src/asan/errors.rs

```diff
@@ -241,24 +243,16 @@ impl AsanErrors {
                     .unwrap();
 
                 #[cfg(target_arch = "x86_64")]
-                let mut cs = Capstone::new()
-                    .x86()
-                    .mode(capstone::arch::x86::ArchMode::Mode64)
-                    .detail(true)
-                    .build()
-                    .expect("Failed to create Capstone object");
-
-                cs.set_skipdata(true).expect("failed to set skipdata");
+                let decoder = InstDecoder::minimal();
 
                 let start_pc = error.pc - 4 * 5;
-                for insn in &*cs
-                    .disasm_count(
-                        unsafe { std::slice::from_raw_parts(start_pc as *mut u8, 4 * 11) },
-                        start_pc as u64,
-                        11,
-                    )
-                    .expect("failed to disassemble instructions")
-                {
+                let insns = disas_count(
+                    &decoder,
+                    unsafe { std::slice::from_raw_parts(start_pc as *mut u8, 4 * 11) },
```

Comment:
perhaps we should increase the `4` here to `16` (which is close enough to the limit of a single instruction IIRC), and then only display the first 11 instructions?

Reply:
ok